Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasketViewController.m

```diff
@@ -31,15 +35,51 @@ - (IBAction)nextButtonTapped:(id)sender {
     //switch tab controller to index 0
 }
 
-/*
-#pragma mark - Navigation
+- (IBAction)headerImageButtonTapped:(id)sender {
+        UIImagePickerController *imagePickerVC = [UIImagePickerController new];
+        imagePickerVC.delegate = self;
+        imagePickerVC.allowsEditing = YES;
+        BOOL cameraSourceAvail = [UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera];
+        BOOL photoLibAvail = [UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypePhotoLibrary];
+        if (cameraSourceAvail && photoLibAvail) {
```

Comment:
IIRC you already have this exact same logic to present an image picker or action sheet depending on the source types available in the profile screen, you should encapsulate and reuse it instead of duplicating the code.